src/RegisterMismatch.jl

```diff
@@ -19,6 +19,20 @@ export
     mismatch_apertures,
     mismatch_apertures!
 
+const INNER_THREADING = Ref{Bool}(true)
+allow_inner_threading!(state::Bool) = (INNER_THREADING[] = state)
```

Comment:
Is this part of the public API? If it is, it should be documented and maybe also exported.

Reply:
I added docstrings but didn't export as I think it's ambiguous outside of this package